content/sdk/10.js/00.ethers/10.v5/10.types.md

```diff
@@ -259,103 +530,193 @@ Deprecated in favor of `[LogProof](#logproof)`
 
 Interface representation of message proof.
 
-- `id`: `number` - Identifier of the log within the transaction.
-- `proof`: `string[]` - Each element represents a piece of the proof for the specified log.
-- `root`: `string` - Root hash of the proof, anchoring it to a specific state in the blockchain.
+::collapsible
+  ::field{name="id" type="number"}
+  Identifier of the log within the transaction.
+  ::
+  ::field{name="proof" type="string[]"}
+  Each element represents a piece of the proof for the specified log.
+  ::
+  ::field{name="root" type="string"}
+  Root hash of the proof, anchoring it to a specific state in the blockchain.
+  ::
+::
 
 ## `Network`
 
 Enumerated list of networks and their ids.
 
-- Mainnet = `1`
-- Ropsten = `3`
-- Rinkeby = `4`
-- Sepolia = `6`,
-- localhost = `9`,
-- EraTestNode = `10`,
+::collapsible
+  ::field{name="Mainnet" type="number" defaultValue="1"}
+  ::
+  ::field{name="Ropsten" type="number" defaultValue="3"}
+  ::
+  ::field{name="Rinkeby" type="number" defaultValue="4"}
+  ::
+  ::field{name="Sepolia" type="number" defaultValue="6"}
+  ::
+  ::field{name="localhost" type="number" defaultValue="9"}
+  ::
+  ::field{name="EraTestNode" type="number" defaultValue="10"}
+  ::
+::
```

Comment:
Same with these, they are specific values to pass in, not properties. Please double check through all of the changes you made to ensure you edited properties and not values for some sections.